roles/opensrp/tasks/compile.yml

```diff
@@ -48,6 +48,11 @@
   with_items:
   - "context"
 
+- name: Set Compiler Java home
+  environment:
```

Comment:
Instead of having an exclusive task for this, you should add the environment map to the "Compile the OpenSRP WAR" task.